scripts/check-tsc.ts

```diff
@@ -0,0 +1,22 @@
+import { execSync, type SpawnSyncReturns } from "node:child_process";
+import { findDirWithFileRecursive } from "./find.js";
+
+async function main(): Promise<void> {
+  for (const project of await findDirWithFileRecursive("tsconfig.json"))
+    try {
+      console.log(`🔄 ${project}`);
+      execSync("npx tsc --noEmit", {
+        cwd: project,
+        stdio: "inherit",
+      });
+      console.log(`☑️ ${project}`);
```

Comment:
I don't know why, but at least for me the text sticks to the emoji unless I add two spaces:

```suggestion
      console.log(`☑️  ${project}`);
```

Reply:
Hmm only for `☑️` or for `🔄` as well?